spec/open_api_spec.rb

```diff
@@ -73,7 +73,19 @@
       it "with a random prefix" do
         expect(ENV["PATH_PREFIX"]).not_to be_nil
         expect(ENV["APP_NAME"]).not_to be_nil
-        expect(api_v1x2_requests_url(:only_path => true)).to eq("/#{ENV["PATH_PREFIX"]}/#{ENV["APP_NAME"]}/v1.2/requests")
+        uri = "/#{ENV["PATH_PREFIX"]}/#{ENV["APP_NAME"]}/v1.2/requests"
+        expect(api_v1x2_requests_url(:only_path => true)).to eq(URI.encode(uri))
```

Comment:
I was hoping we could use `ERB::Util.url_encode(url)` [as I've used in the past](https://github.com/ManageIQ/manageiq-automation_engine/blob/bf855b9c7073b355ec75fb9bfe675cc6a791b539/lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_uri.rb#L12) but that had issues as well.